src/actions/tokens.js

```diff
@@ -0,0 +1,42 @@
+import { createActions } from 'redux-actions'
+import { wallet } from '../utils/wallet'
+import { getLikelyContracts, getMetadata, getBalanceOf } from '../utils/tokens'
+
+const WHITELISTED_CONTRACTS = (process.env.TOKEN_CONTRACTS || 'berryclub.ek.near,farm.berryclub.ek.near,wrap.near').split(',');
+
+export const handleGetTokens = () => async (dispatch, getState) => {
+    const { accountId } = getState().account
+
+    await dispatch(tokens.likelyContracts.get(accountId))
+
+    let contracts = [...new Set([...getState().tokens.likelyContracts, ...WHITELISTED_CONTRACTS])].reduce((x, contract) => ({
+        ...x,
+        [contract]: { contract }
+    }), {})
+
+    dispatch(tokens.tokensDetails.set(contracts))
```

Comment:
not sure why this action is needed vs doing same work in the reducer for `tokens.likelyContracts.get`

Reply:
Right, done